sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryUtilsTest.java

```diff
@@ -422,4 +424,30 @@ public void testToBeamRow_array_row() {
     Row beamRow = BigQueryUtils.toBeamRow(ARRAY_ROW_TYPE, BQ_ARRAY_ROW_ROW);
     assertEquals(ARRAY_ROW_ROW, beamRow);
   }
+
+  @Test
+  public void testToBeamRow_repeated_row() {
```

Comment:
Thanks for adding this test. It's much more clear now on the change. So this change extends a general support for `ARRAY<T>`.

Optional, it would be great if you could also test `ARRAY<ARRAY<T>>`

Reply:
Great idea! I added that test as well